.github/workflows/manual-safe-tools-client.yml

```diff
@@ -20,6 +20,18 @@ jobs:
       - uses: ./.github/actions/setup-node-modules-cache
         with:
           yarn_lock_md5: ${{ hashFiles('yarn.lock') }}
+      - name: Set up env
```

Comment:
This is copied from the web client config: https://github.com/cardstack/cardstack/blob/main/.github/workflows/manual-web-client.yml#L23